aiida/tools/importexport/dbimport/backends/common.py

```diff
@@ -22,6 +25,34 @@
 MAX_GROUPS = 100
 
 
+def _copy_node_repositories(*, uuids_to_create: List[str], reader: ArchiveReaderAbstract):
+    """Copy repositories of new nodes from the archive to the AiiDa profile.
+
+    :param uuids_to_create: the node UUIDs to copy
+    :param reader: the archive reader
+
+    """
+    if not uuids_to_create:
+        return
+    IMPORT_LOGGER.debug('CREATING NEW NODE REPOSITORIES...')
+    with get_progress_reporter()(total=1) as progress:
```

Comment:
Ok, so this progress reporter is meant to be overwritten by the callback, right?
Perhaps add a comment to that effect.

Reply:
yes the callback manipulates the progress reporter.
I thought this would be clear in the `_callback` function below? but if not can add a comment

Reply:
I've added a comment though